pocket_lab/pl_gui_views.c

```diff
@@ -1060,7 +1105,7 @@ int32_t pl_gui_create_attributes_view(lv_obj_t *parent,
 				      struct pl_gui_init_param *param)
 {
 	int32_t ret;
-	char dropdown_list[100];
+	char dropdown_list[200];
```

Comment:
What is the reason behind this change?

Reply:
I have mentioned it as comment on the PR. I would move it to the commit message for a better clarity.

Reply:
My bad. I did not notice this